### ISSUE TITLE
What is the password?

### ISSUE THREAD

**Issue by dinhanhx** [dinhanhx/Information-Security-USTH-2020#1]
The password has 3 letters. That the hint. is it French? I don't know.

**Comment by Huy-Ngo** [comment]
Hmm... there are only 17576 possibilities for the secret key; not very secure, isn't it?

**Comment by dinhanhx** [comment]
Yeah but the problem is that you have to figure out the algorithms I used.

**Comment by dinhanhx** [comment]
Well for anyone who want the answer, head to this [link](https://drive.google.com/drive/folders/1UUlHahZu9dGSEVMjM7UoNxpcIvxas0qt?usp=sharing)
